static/css/custom.css

```diff
@@ -556,4 +556,35 @@ dt {
 }
 .quire-navbar-page-controls {
   transform: rotate(180deg);
-}
\ No newline at end of file
+}
+
+/* GLOSSARY
+============================================================================= */
+.content ul.quire-glossary__key {
+  margin-left: 0;
+  text-align: center;
+  display: table;
+  margin-bottom: 2rem;
+}
+.quire-page__content .container .content ul.quire-glossary__key li::before {
+  content: "";
+}
+.quire-page__content .container .content ul.quire-glossary__key li {
+  margin: .5rem .75rem;
+  display: inline-block;
+}
+.quire-page__content .content dl.quire-glossary {
+  margin: 2rem 0;
+}
+.quire-page__content .content dl.quire-glossary dt {
+  display: inline;
+  margin-right: .3rem
+}
+.quire-page__content .content dl.quire-glossary dd {
+  display: inline;
```

Comment:
With the glossary terms bold, we don't need the large margin between the term and the definition. I'd add `margin-left: .2;` here.

Reply:
Hmm, @geealbers I believe adding this line created a large indent. See screenshots. I used inspect to play around with isolating the problem/fixing it. 

However, I've since removed `margin-left: .2;` from the custom CSS and it's still appearing with an indent in my local preview. 

<img width="1160" alt="Screen Shot 2021-04-01 at 2 39 53 PM" src="https://user-images.githubusercontent.com/62075223/113357100-3e516b00-92f8-11eb-987e-3cbae7c18d40.png">
<img width="1162" alt="Screen Shot 2021-04-01 at 2 40 07 PM" src="https://user-images.githubusercontent.com/62075223/113357107-414c5b80-92f8-11eb-8cbd-c7cac501e1ee.png">

<img width="681" alt="Screen Shot 2021-04-01 at 2 37 41 PM" src="https://user-images.githubusercontent.com/62075223/113357712-478f0780-92f9-11eb-8c14-4ef3ff08106b.png">

Reply:
Oh, sorry, try `margin-left: .2em;` (I forgot the `em` units the first time). This should override the `margin-left: 2em;` that's in the stylesheets currently and that's causing that large indent.

Reply:
Perfect. That worked!